contracts/ForceMove.sol

```diff
@@ -129,15 +126,13 @@ contract ForceMove {
             variableParts
         );
 
-        channelStorageHashes[channelId] = keccak256(
-            abi.encode(
-                ChannelStorage(
-                    largestTurnNum,
-                    now + fixedPart.challengeDuration,
-                    supportedStateHash,
-                    challenger,
-                    keccak256(abi.encode(variableParts[variableParts.length - 1].outcome))
-                )
+        channelStorageHashes[channelId] = _getHash(
```

Comment:
I am wondering if we still want to use 'hash' to refer to the data that is stored on chain? It is no longer the output of `keccak256`.

How about something like the following: the mapping is named `storageData`, and `_getHash` becomes `_encodeStorageDatum` etc?